test_evm_eth_compliance/test-vectors/02_GeneralStateTests_status.md

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
Can we move reports and documentation (except for the README) to a `docs` package?